c_src/device/drm.c

```diff
@@ -532,43 +534,43 @@ int init_egl( const device_opts_t* p_opts, device_info_t* p_info )
 
   g_egl_data.display = eglGetDisplay(gbm.dev);
   if (!eglInitialize(g_egl_data.display, &g_egl_data.major_version, &g_egl_data.minor_version)) {
-    fprintf(stderr, "failed to initialize egl\n");
+    log_error("failed to initialize egl");
     return -1;
   }
 
-  // fprintf(stderr, "Using display %p with EGL version %d.%d\n", g_egl_data.display, g_egl_data.major_version, g_egl_data.minor_version);
-  // fprintf(stderr, "EGL Version \"%s\"\n", eglQueryString(g_egl_data.display, EGL_VERSION));
-  // fprintf(stderr, "EGL Vendor \"%s\"\n", eglQueryString(g_egl_data.display, EGL_VENDOR));
-  // fprintf(stderr, "EGL Extensions \"%s\"\n", eglQueryString(g_egl_data.display, EGL_EXTENSIONS));
+  // log_info("Using display %p with EGL version %d.%d", g_egl_data.display, g_egl_data.major_version, g_egl_data.minor_version);
+  // log_info("EGL Version \"%s\"", eglQueryString(g_egl_data.display, EGL_VERSION));
+  // log_info("EGL Vendor \"%s\"", eglQueryString(g_egl_data.display, EGL_VENDOR));
+  // log_info("EGL Extensions \"%s\"", eglQueryString(g_egl_data.display, EGL_EXTENSIONS));
```

Comment:
Similar to some other deadcode I've seen...perhaps this could be behind a `#ifdef` instead? If not, we should delete it.

Reply:
removed